src/Backend/Modules/Groups/Entity/GroupModuleRight.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Backend\Modules\Groups\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * This is the GroupModuleRight Entity
+ *
+ * @author Mathias Dewelde <mathias@dewelde.be>
+ *
+ * @ORM\Entity
+ * @ORM\Table(name="GroupModuleRight")
```

Comment:
You should use the old database table name here. So nothing breaks in other modules.
After the full Doctrine switch is done, we can remove this line in all Entities.
```php
 * @ORM\Table(name="groups_rights_modules")
```